cmd/capture.go

```diff
@@ -19,7 +19,7 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/alegrey91/harpoon/internal/captor"
+	"github.com/alegrey91/harpoon/internal/ebpf/probesfacade/captor"
 	"github.com/alegrey91/harpoon/internal/writer"
 	"github.com/spf13/cobra"
```

Comment:
Cobra should be in a separate block before the packages of the module

Reply:
I think is better open another issue for refactoring all these things

Reply:
may be a good PR to fix all the errors reported by `golangci-lint` that you are going to add